lib/cuckoo/core/rooter.py

```diff
@@ -12,9 +12,8 @@
 router_cfg = Config("routing")
 log = logging.getLogger(__name__)
 
-vpns = {}
-socks5s = {}
-
+vpns = dict()
+socks5s = dict()
```

Comment:
FYI for future if is already declared, don't change that as ruff and other will ask to restore from dict() to {}